src/GRA.Controllers/MappingProfile.cs

```diff
@@ -29,6 +29,9 @@ public MappingProfile()
                     Value = _.Color
                 })))
                 .ReverseMap();
+            CreateMap<AvatarBundle, AvatarBundleJsonModel.AvatarBundle>()
```

Comment:
Is it possible for us to do this without using AutoMapper?

Reply:
When I just try serializing the object to JSON and executed it, it throws a 

Uncaught SyntaxError: Unexpected token a in JSON at position 177
    at JSON.parse (<anonymous>)
    at Function.i.parseJSON

And breaks the Avatar page.

Reply:
I think @iafb might have better insight as to why we have to use the AutoMapper, from a more technical perspective

Reply:
We'll talk to him.